iati/core/rulesets.py

```diff
@@ -232,13 +227,17 @@ def __init__(self, xpath_base, case):
         super(RuleAtLeastOne, self).__init__(xpath_base, case)
 
     def is_valid_for(self, dataset):
-        """Check `dataset.xml_tree` has at least one instance of a given case for an Element.
+        """Check dataset has at least one instance of a given case for an Element.
 
         Args:
-            dataset.xml_tree: an etree created from an XML dataset.
+            dataset (iati.core.Dataset): An IATI Dataset object.
 
         Returns:
-            Boolean value that changes depending on whether the case is found in the dataset.xml_tree.
+            bool: Changes depending on whether the case is found in the dataset.
+
+        Raises:
+            AttributeError: When an argument is given that is not a dataset object.
+            XPathEvalError: When no valid xpath argument is given.
```

Comment:
`no valid xpath argument` - The function doesn't take an argument called `xpath`. As such, it is unclear what this is referring to.